.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
     "env": {
-        "browser": true,
+        "node": true,
```

Comment:
Fixed ESLint warning `'process' is not defined` for `process.env` within  `scripts/prod.config.js`
Thanks to this [StackOverflow](https://stackoverflow.com/questions/50894000/eslint-process-is-not-defined) thread